src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTable/MyProfileCounterpartiesTable.tsx

```diff
@@ -68,8 +68,8 @@ const MyProfileCounterpartiesTable = ({
     if (!isFetching && data.length === 0) {
         if (searchValue === '') return <MyProfileCounterpartiesEmpty />;
         return (
-            <Text weight='bold'>
-                <Localize i18n_default_text='There are no matching name' />
+            <Text className='pt-12' weight={isMobile ? 'normal' : 'bold'}>
```

Comment:
may be we should ask the design team why do we have it as normal in mobile and bold in desktop 😅

Reply:
yeah 😅 lots of weird inconsistencies